dll/win32/wdmaud.drv/mmixer.c

```diff
@@ -776,7 +776,35 @@ WdmAudGetWavePositionByMMixer(
     IN  struct _SOUND_DEVICE_INSTANCE* SoundDeviceInstance,
     IN  MMTIME* Time)
 {
-    /* FIXME */
+    PSOUND_DEVICE SoundDevice;
+    MMDEVICE_TYPE DeviceType;
+    MIXER_STATUS Status;
+    MMRESULT Result;
+    DWORD Position;
+
+    Result = GetSoundDeviceFromInstance(SoundDeviceInstance, &SoundDevice);
+
+    if ( ! MMSUCCESS(Result) )
+    {
+        return TranslateInternalMmResult(Result);
+    }
```

Comment:
```suggestion
    if (!MMSUCCESS(Result))
        return TranslateInternalMmResult(Result);
```
see coding guidelines. No spaces-padding within curved braces.

Reply:
Committed, thanks.